terraform/aio/templates/userdata.cfg.tpl

```diff
@@ -2,3 +2,11 @@
 # Don't automatically mount ephemeral disk
 mounts:
   - [/dev/vdb, null]
+# WORKAROUND: internal DNS missing from SMS lab.
+runcmd:
+  - 'echo "10.0.0.34 pelican pelican.service.compute.sms-lab.cloud" >> /etc/hosts'
+  - 'echo "10.205.3.187 pulp-server pulp-server.internal.sms-cloud" >> /etc/hosts'
+# Configure SSH keys here, to avoid creating an ephemeral keypair.
+# This means only the instance needs to be cleaned up if the destroy fails.
+ssh_authorized_keys:
```

Comment:
IIRC this didn't work for me in the Ubuntu job, unsure which user got the key.

Reply:
Cheers, I didn't check this as I only saw the ubuntu PR late and thought I'd adopt the method from there.

Reply:
Fixed with https://github.com/stackhpc/stackhpc-kayobe-config/pull/155/commits/ff692fa94a46d675a7ffc114d9c464c914192db7